tests/baselines/reference/moduleAugmentationGlobal5.js

```diff
@@ -32,3 +32,32 @@ require("B");
 //// [f3.d.ts]
 import "A";
 import "B";
+
+
+//// [DtsFileErrors]
+
+
+f3.d.ts(1,8): error TS2307: Cannot find module 'A' or its corresponding type declarations.
+f3.d.ts(2,8): error TS2307: Cannot find module 'B' or its corresponding type declarations.
+
+
+==== f3.d.ts (2 errors) ====
+    import "A";
+           ~~~
+!!! error TS2307: Cannot find module 'A' or its corresponding type declarations.
+    import "B";
+           ~~~
+!!! error TS2307: Cannot find module 'B' or its corresponding type declarations.

```

Comment:
I think this shows an extra bug that was here before this PR but one that slipped through the cracks as those errors were not surfaced. 

Those modules are declared by other files and the original `.ts` file here referenced them using tripleslash directives. Those directives were pruned from the resulting `.d.ts` so the declaration set can't locate those modules now.